src/Centreon/Tests/Application/DataRepresenter/BulkTest.php

```diff
@@ -42,8 +42,10 @@
 
 class BulkTest extends TestCase
 {
-
-    public function testJsonSerialize()
+    /**
+     * @return void
+     */
```

Comment:
docBloc should describe what the test does
```suggestion
```

Reply:
done